sdk/identity/identity/src/credentials/managedIdentityCredential/fabricMsi.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AccessToken, GetTokenOptions, RequestPrepareOptions } from "@azure/core-http";
+import { MSI } from "./models";
+import { credentialLogger } from "../../util/logging";
+import { IdentityClient } from "../../client/identityClient";
+import { msiGenericGetToken } from "./utils";
+
+const logger = credentialLogger("ManagedIdentityCredential - Fabric MSI");
+
+function expiresInParser(requestBody: any): number {
+  // Parses a string representation of the seconds since epoch into a number value
+  return Number(requestBody.expires_on);
+}
+
+function prepareRequestOptions(resource: string, clientId?: string): RequestPrepareOptions {
+  const queryParameters: any = {
+    resource,
+    "api-version": "2019-07-01"
```

Comment:
According to the [documentation](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-identity-service-fabric-app-code#acquiring-an-access-token-using-rest-api), only the preview version is available
```suggestion
    "api-version": "2019-07-01-preview"
```